tooling/strace_analysis.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+# Before executing this script, strace output files need to be generated
```

Comment:
Need to add the Apache license here, eg: https://github.com/adoptium/temurin-build/blob/c9415f1e1ec0d85d518e5ca1f9818dc8b02882e5/tooling/repro_common.sh#L3

Reply:
Done